index.html

```diff
@@ -0,0 +1,233 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" type="text/css" href="style.css">
+
+</head>
+<body>
+  
+
+    <header class="nav">
```

Comment:
- [ ] Please remove some white spaces in your Html file because it is the best programming practice. Unnecessary white spaces makes your code harder to read 😃 😎